build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.9.0'
+    ext.kotlin_version = '1.8.22'
```

Comment:
Is it possible that the AGP 7.4.1 needs to be upgraded in order for this to not be downgraded?

Reply:
Yeah, I had it down to 1.7.x as part of the process for a while and then ran into other issues. It'll change again when I do the Gradle 8 change.